### PR TITLE
Subaru: Add missing FW versions for Forester 2019 and Impreza 2020

### PR DESCRIPTION
**Pull request by martinl**
This PR is a collection of FPv2 fw versions for Forester 2019-21 and Crosstrek/Impreza 2020-21 submitted by users, related to removal of both models FPv1 fingerprints (https://github.com/commaai/openpilot/pull/24195)

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
@sshane let's build a route list of at least a few routes from each of those devices and run `selfdrive/debug/test_fw_query_on_routes.py` to verify they will all fingerprint correctly

**Comment by sshane** [comment]
In this route, a Forester would fail to fingerprint as it only reported 4 fw versions, and we exclude 1 type (fwdCamera) and according to @martinl can't add the fwdRadar fw. Previously this fingerprinted via CAN

```python
'carFw': [{'address': 2000, 'ecu': 'fwdRadar', 'fwVersion': b'\xf1\x00\x00\x00\x02', 'subAddress': 0},
          {'address': 1988, 'ecu': 'fwdCamera', 'fwVersion': b'\xf1\x00\xac\x02\x00', 'subAddress': 0},
          {'address': 2016, 'ecu': 'engine', 'fwVersion': b'\xf1\x00\xa2\x10\n', 'subAddress': 0},
          {'address': 2017, 'ecu': 'transmission', 'fwVersion': b'\xf1\x00\xa4\x10@', 'subAddress': 0}],
```

**Comment by sshane** [comment]
Checked 311 unique routes for the cars from your commits, breakdown:

```python
# BY CAR
SUBARU IMPREZA SPORT 2020: 54 routes (6 failed fingerprinting)
SUBARU FORESTER 2019: 257 routes (45 failed fingerprinting)

# BY DONGLE ID
a2af7643da1d4017: 29 routes (6 failed fingerprinting)
c2f528d49fe19d53: 48 routes (0 failed fingerprinting)
0f33f7a9d1949fa6: 25 routes (0 failed fingerprinting)
f6e178e18ba27ca1: 36 routes (3 failed fingerprinting)
6fcb17b9ef924548: 80 routes (0 failed fingerprinting)
a447729c1d15ff89: 93 routes (42 failed fingerprinting)
```

For most of `SUBARU FORESTER 2019`'s failures to fingerprint, the ecus `esp`, `eps`, and `fwdCamera` were missing

File: selfdrive/car/subaru/values.py [hunk 1]
```diff
@@ -164,58 +164,72 @@ class SubaruCarInfo(CarInfo):
...
       b'\xe7\xf6B0\000',
       b'\xe7\xf5D0\000',
+      b'\xf1\x00\xd7\x10@',
+      b'\xe6\xf5D0\x00',
     ],
   },
   CAR.FORESTER: {
     (Ecu.esp, 0x7b0, None): [
-      b'\xa3 \030\024\000',
+      b'\xa3 \x18\x14\x00',
```
Comment (sshane) [inline selfdrive/car/subaru/values.py:202]: same fw

File: selfdrive/car/subaru/values.py [hunk 2]
```diff
@@ -164,58 +164,72 @@ class SubaruCarInfo(CarInfo):
...
       b'\xa3  \024\000',
       b'\xa3 \031\024\000',
-      b'\xa3  \024\001',
+      b'\xa3  \x14\x01',
```
Comment (sshane) [inline selfdrive/car/subaru/values.py:205]: same fw

**Comment by sshane** [comment]
Thanks for the PR!

**Pull request merged**